types/k6/browser/index.d.ts

```diff
@@ -393,19 +393,16 @@ export interface ScreenshotOptions {
  * - `mutation` - use a mutation observer
  * - `interval` - use a polling interval
  */
-export type PollingMethod = "raf" | "mutation" | "interval";
+export type PollingMethod = number | "raf" | "mutation";
```

Comment:
Out of curiosity; what is `"mutation"` used for? I couldn't find it in our docs, nor in Playwright docs 🤔

Reply:
It works with [mutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) for changes in the DOM and calls the observer when anything changes.

Reply:
I've updated it in the TD: 76e4a3ddfa8d02eecfb6794272f652269d61a1b9

Reply:
Thanks! 🙌🏻